models/tridimensional/cas9_PAM_scoring.py

```diff
@@ -0,0 +1,86 @@
+"""
+IMPORTANT
+Modify script to point to the location of iGEM Dropbox on your computer. Thanks! :)
+
+- Kevin
+
+"""
+
+
+from time import time
+import os
+from rosetta import *
+
+# Navigate to where the PDBs are located
+os.chdir("/home/aruu/Documents/Dropbox/Waterloo iGEM 2015") # Change to location of your iGEM Dropbox
+os.chdir("Math Modelling/cas9_modification")
+
+# Initialize PyRosetta
+init()
+
+# Variations to account for
+dna_nts = "acgt"
+programs = ["Chimera", "3DNA"]
+
+
+# If we wanna parallelize+distribute this, probably easiest just for each person to
+# hard code what range they are responsible for (to avoid everyone processing the
+# same 4 tasks, no way to tell what has already been started by someone else)
+
+for i in range(0, 64):
+	# Creates a three nucleotide PAM sequence, aaa then, aac, aag, etc.
+	variant = dna_nts[i / 16] + dna_nts[i / 4 % 4] + dna_nts[i % 4]
```

Comment:
0 to 15 /16 will return 0, 16to 31  / 16 will return 1, etc

python naturally floors ints